docker-multinode/common.sh

```diff
@@ -123,40 +125,12 @@ kube::multinode::detect_lsb() {
   kube::log::status "Detected OS: ${lsb_dist}"
 }
 
-# Start a docker bootstrap for running etcd and flannel
-kube::multinode::bootstrap_daemon() {
```

Comment:
OK what about starting etcd and flannel with parameter `-H ${BOOTSTRAP_DOCKER_SOCK}` and all network configuration? Should we  create additional code or some flag to distinguish between CNI or standard network ?